single_tech_samples/fabric/fabric_ci_cd/README.md

```diff
@@ -57,12 +57,13 @@ Refer to the [know issues, limitations, and workarounds](./issues-limitations-an
 
 Here are the steps to use the bootstrap script:
 
-1. Clone the repository:
+1. Clone the repository and get the subscription information for your environment variables:
 
     ```bash
     # Optional
     az login --tenant "<tenant_id>"
```

Comment:
When running this command, you probably will be prompted for a number that refers the subscription ID - just maybe make a note referring that if you are prompted for the number, you don't need the az account set command.